components/com_users/controllers/registration.php

```diff
@@ -63,7 +63,10 @@ public function activate()
 
 		if (!$userIdToActivate)
 		{
-			JError::raiseError(403, JText::_('COM_USERS_ACTIVATION_TOKEN_NOT_FOUND'));
+			$loginUrl = 'index.php?option=com_users&view=login';
+
+			$this->setMessage(JText::_('COM_USERS_ACTIVATION_TOKEN_NOT_FOUND'));
+			$this->setRedirect(JRoute::_($loginUrl, false));
```

Comment:
```suggestion
			$this->setRedirect(JRoute::_('index.php?option=com_users&view=login', false));
```
Like this and remove line 66.